### PR TITLE
Bye-bye Orange Bubbles!

### PR DESCRIPTION
**Pull request by seancribbs**
This pull-request accomplishes several things I discussed with @joedevivo and @cmeiklejohn today, namely:
1. Orange bubbles indicating mixed results are no more. The bubble on the scorecard now shows the result of the **latest** recorded test.
2. **Scorecard names are normalized** to a version string more closely resembling our release process (grouping whole versions and release-candidates together). **Test results still record the entire version string** for comparison purposes, meaning we can see whether given builds fix tests before cutting a new tag.

Example:

screenshot

### CODE REVIEW

File: assets/javascripts/app/views.js [hunk 1]
```diff
@@ -36,21 +36,21 @@ GiddyUp.ScorecardSubcellView = Ember.View.extend({
...
     else
-      return 'badge-warning';
+      return 'badge-important';
   }.property('content.status'),
-  percent: function(){
-    var status = this.get('content.status');
-    return status.percent.toFixed(1).toString() + "%";
+  title: function(){
+    var percent = this.get('content.status.percent');
+    if(percent != null || percent != undefined)
```
Comment (cmeiklejohn) [inline assets/javascripts/app/views.js]: !==?

File: assets/javascripts/app/templates/test_results_collection_item_view.hbs [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {{#with view.content}}
   {{#if success}}
-  <a {{action showTestResult this href=true}}><span class="label label-success">Success</span> Test Result: {{id}}</a>
+  <a {{action showTestResult this href=true}}><span class="label label-success">Success</span> Test Result: {{id}} {{long_version}}</a>
   {{/if}}
   {{#if failure}}
-  <a {{action showTestResult this href=true}}><span class="label label-important">Failure</span> Test Result: {{id}}</a>
+  <a {{action showTestResult this href=true}}><span class="label label-important">Failure</span> Test Result: {{id}} {{long_version}}</a>
```
Comment (cmeiklejohn) [inline assets/javascripts/app/templates/test_results_collection_item_view.hbs]: Should we nuke the ID?  Doesn't seem of any value, does it?

File: assets/javascripts/app/models.js [hunk 3]
```diff
@@ -124,29 +125,50 @@ GiddyUp.ScorecardCell = Ember.Object.extend({
...
+  }.property('total', 'numPassing'),
+
+  percent: function(){
+    if(this.get('total') === 0)
+      return 0.0;
+    else
+      return (this.get('numPassing') / this.get('total')) * 100;
+  }.property('total', 'numPassing'),
+
+  latestBinding: 'subcell.test_results.firstObject.status'
```
Comment (cmeiklejohn) [inline assets/javascripts/app/models.js:151]: Shouldn't we sort test_results? Or maybe I'm misunderstanding something?

**Comment by cmeiklejohn** [comment]
Otherwise :+1:, looks great.

File: assets/javascripts/app/models.js (hunk 3 above)
Comment (seancribbs) [inline assets/javascripts/app/models.js:151]: Look in the ScorecardSubcell, it is sorted.

**Comment by seancribbs** [comment]
@cmeiklejohn Did I address those comments appropriately?

File: assets/javascripts/app/models.js (hunk 3 above)
Comment (cmeiklejohn) [inline assets/javascripts/app/models.js:151]: I had looked at the wrong test_results.  :+1:

**Comment by cmeiklejohn** [comment]
:+1: :beers:

**Comment by seancribbs** [comment]
@cmeiklejohn (ó ì_í)=óò=(ì_í ò)

**Comment by cmeiklejohn** [comment]
http://www.youtube.com/watch?v=_mSmOcmk7uQ

**Pull request merged**